packages/svelte/src/compiler/legacy.js

```diff
@@ -451,6 +451,7 @@ export function convert(source, ast) {
 			SpreadAttribute(node) {
 				return { ...node, type: 'Spread' };
 			},
+			// @ts-ignore
```

Comment:
was this... failing before...?

Reply:
who the hell knows. legacy === not going to waste brain cycles on it